packages/core/contracts/common/interfaces/AddressWhitelistInterface.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+interface AddressWhitelistInterface {
+    function addToWhitelist(address newElement) external;
```

Comment:
do you not need to label these as `virtual`?

Reply:
ye good point. done!